External/FEXCore/include/FEXCore/fextl/vector.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <fextl/allocator.h>
+
+#include <vector>
+
+namespace fextl {
+  template<class T>
```

Comment:
For the sake of consistency, we might want to keep the allocator template parameter for the custom containers as well, since there's a couple of places we use PMR allocators. Using the same interface for those allows us to `grep` for `std::vector` to ensure we don't use it.

Reply:
Passed through the allocator as a default templated argument. std::pmr interfaces will convert to `fextl::pmr` at some point as well, but they are less of an immediate concern since the allocators those interfaces use are all very specific.